docs/testing-pre-releases.md

```diff
@@ -94,29 +94,16 @@ apt-get install <package name>=<version number>
 yum install <package name>-<version number>
 ```
 
-### Getting kubeadm binaries from a GCS bucket
+### Getting kubeadm binaries from a release bucket
 
-Pre-compiled GA, alpha/beta versions of kubeadm binary are deployed into `gs://kubernetes-release/release/` GCS bucket,
+Pre-compiled GA, alpha/beta versions of kubeadm binary are deployed into `https://dl.k8s.io/release/` release bucket,
 while CI/CD versions are deployed into `gs://k8s-release-dev/ci/` bucket.
```

Comment:
same Q as above for the CI bucket

Reply:
> k8s-release-dev will be tracked separately and isn't meant to be end-user facing (unlike dl.k8s.io), it's meant for contributors to the project

https://github.com/kubernetes/k8s.io/issues/2396#issuecomment-1548048525